### PR TITLE
Migrate Android app to pull list of possible sites from registry via …

### PR DESCRIPTION
**Pull request by ram2012k**
please review

### CODE REVIEW

**Comment by yochannah** [comment]
@ram2012k Sorry for the delay reviewing - I've been away from the office!

This modification works perfectly when I have a new install of the InterMine :tada:, but the list of InterMines doesn't seem to update with the new mines from the registry if you had the app installed before this update. I guess it's stored locally somewhere too - can we purge the old list and/or combine the old and new?

**Comment by ram2012k** [comment]
From the description , I thought only one fetch is needed . 
Yeah we can modify/purge  it .

**Comment by yochannah** [comment]
Sorry - I didn't realise more than the fetch was needed until I tested it myself! :)

**Comment by ram2012k** [comment]
When the app is reopened , the mines would be fetched from the registry and the same should be updated to the local if the registry/internet is not down , else  if internet is not there mines are loaded from the local database . are these changes you want ?

**Comment by yochannah** [comment]
@ram2012k yes, that would be perfect! So the list is always up-to-date if there is internet.

**Comment by ram2012k** [comment]
Please review

**Comment by yochannah** [comment]
This looks better - thank you so much! I'll need to complete a more thorough review, though and it's friday home-time - I'll get back to this next week! :+1:

File: app/src/main/java/org/intermine/app/net/request/get/GetIntermineListRequest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        }
+    }
+
+    public static class LoadIntermineList extends AsyncTask<String, Void , String>{
+        Exception mException = null;
+
+        @Override
+        protected String doInBackground(String... strings) {
+            String result = "";
+            final String urlString = "http://registry.intermine.org/service/instances?mines=all";
```
Comment (yochannah) [inline app/src/main/java/org/intermine/app/net/request/get/GetIntermineListRequest.java]:
```suggestion
            final String urlString = "http://registry.intermine.org/service/instances?mines=prod";
```

**Review by yochannah — APPROVED** [review]
@ram2012k - this seems to work well, thank you so much! I've suggested one tiny change before merge, just to limit the mines to the prod intermines rather than all of the mines including the beta/dev mines. it's using GitHub's new suggest a change feature - I think you just have to click "apply changes" for it? Once that's done I'll merge.

**Comment by ram2012k** [comment]
One more question , what will the search bar searches ?

**Comment by yochannah** [comment]
All other features are mine-specific, but the search bar searched all InterMines at once.

**Comment by ram2012k** [comment]
Why it is not merged yet

**Comment by yochannah** [comment]
I wasn't sure if you hadsomething else you wanted to add based on that question, but I'll assume you don't based on that comment :)  :+1: Thanks for your help @ram2012k!

**Pull request merged**